tpp-app/src/home/CalendarNavigator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import CalendarScreen from './pages/CalendarScreen'
+import YearScreen from './pages/YearScreen'
+
+// Create a navigation stack that can be used to access other pages
+// Stack begins first page at CalendarScreen. This is done so that the global
+// tab bar on the bottom is kept when navigating within pages.
+const Stack = createNativeStackNavigator();
+export default function CalendarNavigator() {
+    return (
+        <Stack.Navigator>
+            <Stack.Screen
+                name={"Calendar"}
+                component={CalendarScreen}
+                options={{
+                    headerShown: false
+                }}
+            />
+            <Stack.Screen
+                name={"Year"}
+                component={YearScreen}
+                options={{
+                    headerShown: false
+                }}
+            />
+        </Stack.Navigator>
```

Comment:
Love the new file structure, very organized! A small suggestion here: since headerShown: false applies to both stack screens, you can just add that styling option directly to <Stack.Navigator> (maybe something like <Stack.Navigator screenOptions={{ headerShown: false }}>). Otherwise, everything else looks great to me!

Reply:
I never realized you could do that, good catch!